pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, runtime: Runtime, group: str, assembly: str, data_path: str,
         self.auto_trigger_build_sync = auto_trigger_build_sync
         self.custom = custom
         self.arches = arches
-        self.in_flight = in_flight
+        self.in_flight = in_flight if in_flight else get_inflight(assembly, group)
```

Comment:
we can have this use self._ocp_version, if we call after when that is present

Reply:
Also let's not do this for a custom assembly

Reply:
prepare-release will call get_assembly_release_date which works with group. its just for convenience

Reply:
https://github.com/openshift-eng/art-tools/pull/316/commits/9398630117ef45b76cd7d1f42fa8c7c2acc04396 should simplify the logic to common